web/src/components/SubscribtionForm.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+
+function SubscriptionForm() {
+	const [subscribedUserEmail, setSubscribedUserEmail] = useState("");
+	const [successfulFetch, setSuccessfulFetch] = useState(false);
+	const [errorMessage, setErrorMessage] = useState("");
+	const url = "/api/user-subscription";
+
+	const fetchUsersSubscription = async () => {
+		try {
+			// Clear previous error messages
+			setErrorMessage("");
+			const postedEmail = await fetch(url, {
+				method: "POST",
+				headers: {
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify({ email: subscribedUserEmail.trim() }),
+			});
+			if (postedEmail.ok) {
+				setSuccessfulFetch(true);
+				setSubscribedUserEmail("");
+			} else {
+				let responseData;
+				try {
+					responseData = await postedEmail.json();
+				} catch {
+					responseData = { message: "Unexpected response format" };
+				}
+				setErrorMessage(responseData.message || "An error occurred");
+				setSuccessfulFetch(false);
+			}
+		} catch (error) {
+			setErrorMessage(error.message);
+			setSuccessfulFetch(false);
+		}
+	};
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		fetchUsersSubscription();
+	};
+
+	return (
+		<>
+			<form onSubmit={handleSubmit}>
+				<input
```

Comment:
You can style the input using the [Ant Input](https://ant.design/components/input) component